app/helpers/path_finder_helper.py

```diff
@@ -19,3 +21,11 @@ def get_path_finder():
 
 
 path_finder = LocalProxy(get_path_finder)
+
+
+def has_path_finder_route(function):
```

Comment:
I wonder if this name is best; looking at the usage it feels more like `requires_full_routing_path`